packages/react-testing/README.md

```diff
@@ -294,6 +295,34 @@ myComponent.triggerKeypath('action.onAction');
 expect(spy).toHaveBeenCalled();
 ```
 
+### <a name="debug"></a> `debug(): string`
+
+Returns a text representation of either the root node, or any element within the mounted graph. `debug()` has two forms, `debug(verbosity?: number = 1, depth?: number)` for simple usage and `debug({verbosity?: number, depth?: number})` for more direct output control. `verbosity` defines the level of expansion that non-scalar props experience, the default value of `1` will expand objects for example one level deep. `depth` defines the number of children printed, by default all children are printed. Typical usage should not require any parameters as the default `verbosity` and `depth` should be appropriate for the majority of inspections.
```

Comment:
I guess most people will just call `debug()` and use the defaults as you say, but I found this pretty hard to parse with such similar arguments (It also doesn't help that the arguments can be passed in a few different ways). Do you think it might be worth simplifying this whole thing?

Reply:
maybe just the options object only then?

Reply:
I think thats the safe bet, yeah